### DISCUSSION TITLE
Consider making this platform more helpful for English learning?

### DISCUSSION THREAD

**Discussion by ChenSongze** [nextai-translator/nextai-translator#1407] [Ideas]
1.Tap a shortcut key to translate sentences where your cursor blinks and display it in-app. Strike 'Enter' to substitute rather than overwrite the entire text.
2.Add a history record, export translated vocabulary for self-importance into tools like Anki, and more.
3.Customize the translation difficulty and lexicon via a prompt-invoking window for a tailored challenge.

Insight: Stepping into a multilingual setting, especially in the fast-paced realms of academia and networking, language learners frequently face a universal and daunting challenge. Even if they have a clear grasp and understanding of specific concepts and expressions in their native tongues, they often struggle to find the accurate equivalent in a foreign language. This "expression barrier" not only chips away at their confidence in communication but also significantly hampers their learning efficiency and social prowess. The commonplace solutions of the moment – such as consulting conventional dictionaries or employing basic translation tools – don't fully satisfy the learner's need for immediacy, accuracy, and authenticity. Hence, the evolution of an instant and reliable input method would spearhead a revolutionary impact on embellishing the fluency and self-assurance in both written and verbal articulations for language learners.

**Comment by yetone** [comment]
‌‌‌‌‌‌‌These are all great suggestions, thank you.
